src/type_conversion/jsproxy.c

```diff
@@ -544,8 +645,54 @@ JsProxy_AsJs(PyObject* x)
   return hiwire_incref(js_proxy->js);
 }
 
+int
+JsException_Check(PyObject* x)
+{
+  return PyObject_TypeCheck(x, (PyTypeObject*)Exc_JsException);
+}
+
+int
+JsException_AsJs(PyObject* err)
+{
+  JsExceptionObject* err_obj = (JsExceptionObject*)err;
+  JsProxy* js_error = (JsProxy*)(err_obj->js_error);
+  return hiwire_incref(js_error->js);
+}
+
 int
 JsProxy_init()
 {
-  return (PyType_Ready(&JsProxyType) || PyType_Ready(&JsBoundMethodType));
+  PyObject* module;
+  PyObject* exc;
+
+  PyExc_BaseException_Type = (PyTypeObject*)PyExc_BaseException;
+  _Exc_JsException.tp_base = (PyTypeObject*)PyExc_Exception;
+
+  module = PyImport_ImportModule("asyncio");
+  if (module == NULL) {
+    goto fail;
+  }
+
+  asyncio_get_event_loop = PyObject_GetAttrString(module, "get_event_loop");
+  if (asyncio_get_event_loop == NULL) {
+    goto fail;
+  }
```

Comment:
That's unrelated to error conversions, isn't it?

Also some `_Py_IDENTIFIER` in the beginning of this file.

Reply:
@oeway Yes, I figured that out and fixed it. I was incorrectly using a C api and it caused crazy undefined behavior. It should have always crashed, I got unlucky with this `Error` vs `new Error` thing.

Reply:
Right, I'm not that great at git and this is bleedover from the `await_jsproxy` PR #880.